packages/grid/src/styled/index.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { IWrapper } from '../interfaces';
+import { getWrapperStyles } from './utils';
+
+
+export * from './body';
+export * from './header';
+export * from './totals';
+
+
+export const Wrapper = styled.div<IWrapper>`
+  width: ${({ width }) => `${width}px`};
+  overflow: hidden;
+  border-radius: 10px;
+  ${({ changingColumns }) => {
+    if (changingColumns === 'move') {
+      return `
```

Comment:
You might to combine "changingColumns === 'move'" and "changingColumns === 'resize'" conditions to one condition and write "cursor: changingColumns === 'move' ? grabbing : w-resize.

Reply:
no, if nor move and not resize - standart cursor